app/code/Magento/Ui/Model/Export/MetadataProvider.php

```diff
@@ -84,7 +85,7 @@ protected function getColumnsComponent(UiComponentInterface $component)
                 return $childComponent;
             }
         }
-        throw new \Exception('No columns found');
+        throw new \Exception('No columns found'); // @codingStandardsIgnoreLine
```

Comment:
Just my curiosity, why is this directive needed?

Reply:
@aleron75  without it throws static code check errors.

Reply:
More specifically, static code warning is: "Direct throw of generic Exception is discouraged. Use context specific instead."

Source: https://github.com/magento/magento-coding-standard/blob/f062ab4e32d4870c4b91deb2b875eb13bbd9c8f9/Magento2/Sniffs/Exceptions/DirectThrowSniff.php#L20

So it should probably be replaced with some other Exception type? Or is that backwards incompatible?

Reply:
@hostep I didn't check all cases when this function used so I prefer to not change it because it is not related to the current issue. The line for ignore I added just ot pass validation.
I can change it to LocalizedException but again cannot speak if it would be compatible will all use cases of this function.